app/helpers/admin/editions_helper.rb

```diff
@@ -279,8 +283,8 @@ def translation_preview_links(edition)
     end
   end
 
-  def archiving_or_unpublishing(edition)
-    edition.unpublishing.unpublishing_reason_id == UnpublishingReason::Archived.id ? 'archiving' : 'unpublishing'
+  def withdrawal_or_unpublishing(edition)
+    edition.unpublishing.unpublishing_reason_id == UnpublishingReason::Withdrawn.id ? 'withdrawal' : 'unpublishing'
```

Comment:
I think it's 6 of one, half dozen of the other. "Edit public explanation for withdrawal" vs "Edit public explanation for withdrawing". Maybe a product decision? @bmwelby?